shimmer/learn/governance/governance-tech.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Governance voting in Shimmer Explained
+description: A technical description of the Governance functionality in Shimmer
+image: /img/learn/governance/vote-graphic04.png
+keywords:
+  - Documentation
+  - Firefly
+  - Hornet INX
+  - Voting
+  - Shimmer
+  - Governance
+---
+
+On-chain governance is important for any decentralized DLT. With Shimmer's Firefly Wallet, Shimmer token holders can participate in the governance of the network. 
```

Comment:
You could use vital or essential instead of important. 

It may be a good idea to add a line explaining why it's important.